src/appleseed/foundation/image/colorspace.h

```diff
@@ -236,10 +237,19 @@ Color<T, 3> linear_rgb_to_hsl(const Color<T, 3>& linear_rgb);
 template <typename T>
 Color<T, 3> ciexyz_to_linear_rgb(const Color<T, 3>& xyz);
 
+template <typename T>
+Color<T, 3> ciexyz_to_linear_rgb(
```

Comment:
On the one hand, these two new functions give a semantic name to the operation; on the other hand, they are perfectly identical and all they do is wrap a matrix * color multiplication (+ clamping)... Let's leave them as is, but it's interesting nevertheless.